build.sbt

```diff
@@ -143,7 +143,9 @@ val instrumentationProjects = Seq[ProjectReference](
   `kamon-lagom`,
   `kamon-finagle`,
   `kamon-aws-sdk`,
-  `kamon-alpakka-kafka`
+  `kamon-alpakka-kafka`,
+  `kamon-http4s-1_0`,
+  `kamon-http4s-0_23`
```

Comment:
I'd add this to `kamon-bundle-dependencies-2-12-and-up` and `kamon-bundle-dependencies-3` too TBH. Seems like we don't currently have a way of including support in the build file for 2.13 bundle and not 2.12 though, so whilst you could also include  `kamon-http4s-1_0` in the scala 3 bundle, don't think we can get it in the 2.13 one. Alas.